tests/unit/slices/filters.ts

```diff
@@ -101,6 +101,44 @@ describe('filter slice reducer works as expected', () => {
     expect(actualState).toEqual(selectedExpectedState);
   });
 
+  it('setFilterOption action is handled properly with a Filter with a NumberRangeValue', () => {
+    const selectedFilter: SelectableFilter = {
+      fieldId: 'id1',
+      matcher: Matcher.Equals,
```

Comment:
does this need to be `Matcher.Between` for a number range?

Reply:
sure, I updated to `Matcher.Between`